src/SimulationRunner.cc

```diff
@@ -716,6 +716,9 @@ void SimulationRunner::Step(const UpdateInfo &_info)
   // Process entity removals.
   this->entityCompMgr.ProcessRemoveEntityRequests();
 
+  // Process components removals
+  this->entityCompMgr.ClearRemovedComponents();
```

Comment:
I noticed that the order between `ClearRemovedComponents` and `ProcessRemoveEntityRequests` is swapped on the server and client.

It probably doesn't matter, right? But just for consistency in case the internal implementation of these functions change in the future, it would be nice to be consistent with the ordering.

Reply:
b913c32